bundles/org.openhab.binding.luxtronikheatpump/README.md

```diff
@@ -207,7 +207,7 @@ The following channels are holding read only values:
 | highPressure | Number:Pressure | x | High pressure |
 | lowPressure | Number:Pressure | x | Low pressure |
 | outputCompressorHeating | Switch | x | Output compressor heating |
-| controlSignalCirculatingPump | Number:Energy | x | Control signal circulating pump |
+| controlSignalCirculatingPump | Number | x | Control signal circulating pump |
```

Comment:
`Number` seems fine to me, but guess `Number:Dimensionless` for percent values would be "more" correct. See https://www.openhab.org/docs/concepts/units-of-measurement.html

Reply:
Agreed. Updated description from https://mw.ait-group.net/files/docs/EN/A0220/83055400.pdf